### PR TITLE
#4508 writing tests for streamEpics

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4508)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/PipelineApiHelper/stream.js [hunk 1]
```diff
@@ -54,34 +53,31 @@ const stream = () => {
       from: [{ name: topic.name }],
     };
     const options = { link, paperApi };
-    updateStreamFromLink(params, options);
+    updateStreamLink(params, options);
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/PipelineApiHelper/stream.js:56]: Thanks 😅

File: ohara-manager/client/src/hooks/streamHooks.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ export const useUpdateStreamAction = () => {
   return useCallback(
     (values, options) => {
       const newValues = { ...values, group };
-      dispatch(actions.updateStream.trigger({ values: newValues, options }));
+      dispatch(actions.updateStream.trigger({ params: newValues, options }));
```
Comment (konekoya) [inline ohara-manager/client/src/hooks/streamHooks.js:74]: For these variable naming. params are for `GET` and `DELETE` whereas `PUT` and `POST` uses _values_. :)

Comment (saivirtue) [reply ohara-manager/client/src/hooks/streamHooks.js:74]: Oh, my bad...
I'll rename all the related files, thanks!

Comment (konekoya) [reply ohara-manager/client/src/hooks/streamHooks.js:74]: No problem. I learned this last time when talking with @eechih ...

**Pull request merged**